elastic/kibana
# Skip malformed actions during bulk action scheduling

Resolves: #189690

This PR skips only the malformed actions rather than throwing error which blocks execution of the valid actions as well.

### To verify:

- Create a pre-configured connectors with below config in the kibana.dev.yml.
Notice that slackWebhook doesn't have any `secrets` config.
```
xpack.actions.preconfigured:
  preconfigured-server-log:
    name: preconfigured-server-log
    actionTypeId: .server-log
  slackWebhook:
    name: preconfigured-slack-webhook
    actionTypeId: .slack    
```

- Create a rule (e.g. always firing) with 2 actions, one with`preconfigured-server-log` and one with `slackWebhook`
- Let the rule run and observe the actions on your terminal.
- There should be an error from the slackWebhook whereas the `.server-log` action still works as expected.
